ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteExporter.java

```diff
@@ -0,0 +1,1072 @@
+     * to do the specific handling
+     *
+     * @param ucteNetwork The target UcteNetwork
+     * @param line The TieLine we want to convert
+     * @see UcteExporter#createXnodeFromTieLine(UcteNetwork, UcteNodeCode, Line)
+     * @see UcteExporter#createTieLineWithGeneratedIds(UcteNetwork, Line)
+     */
+    private void convertTieLine(UcteNetwork ucteNetwork, Line line) {
+        if (isUcteTieLineId(line)) {
+            MergedXnode mergedXnode = line.getExtension(MergedXnode.class);
+            String id1 = mergedXnode.getLine1Name().substring(0, 8); //first node line 1
+            String id2 = mergedXnode.getLine1Name().substring(9, 17); //second node line 1
+            String id3 = mergedXnode.getLine2Name().substring(0, 8); //first node line 2
+            String id4 = mergedXnode.getLine2Name().substring(9, 17); //second node line 2
+
+            Optional<UcteNodeCode> optUcteNodeCode1 = parseUcteNodeCode(id1);
+            Optional<UcteNodeCode> optUcteNodeCode2 = parseUcteNodeCode(id2);
+            Optional<UcteNodeCode> optUcteNodeCode3 = parseUcteNodeCode(id3);
+            Optional<UcteNodeCode> optUcteNodeCode4 = parseUcteNodeCode(id4);
+
+            if (optUcteNodeCode1.isPresent() && optUcteNodeCode2.isPresent() && optUcteNodeCode3.isPresent() && optUcteNodeCode4.isPresent()) {
+                ArrayList<UcteNodeCode> ucteNodeCodeList = new ArrayList<>();
+                ucteNodeCodeList.add(optUcteNodeCode1.get());
+                ucteNodeCodeList.add(optUcteNodeCode2.get());
+                ucteNodeCodeList.add(optUcteNodeCode3.get());
+                ucteNodeCodeList.add(optUcteNodeCode4.get());
+
+                ucteNodeCodeList.forEach(ucteNodeCode ->
+                        createXnodeFromTieLine(ucteNetwork, ucteNodeCode, line)
+                );
+
+                String elementName1 = line.getProperties().getProperty(UcteImporter.ELEMENT_NAME_PROPERTY_KEY + "_1", null);
+                String elementName2 = line.getProperties().getProperty(UcteImporter.ELEMENT_NAME_PROPERTY_KEY + "_2", null);
+
+                UcteElementId ucteElementId1 = new UcteElementId(ucteNodeCodeList.get(0), ucteNodeCodeList.get(1), line.getId().charAt(18));
+                UcteElementId ucteElementId2 = new UcteElementId(ucteNodeCodeList.get(2), ucteNodeCodeList.get(3), line.getId().charAt(40));
+
+                UcteLine ucteLine1 = new UcteLine(ucteElementId1, UcteElementStatus.REAL_ELEMENT_IN_OPERATION, (float) line.getR() / 2,
+                        (float) line.getX() / 2, (float) line.getB1(), (int) line.getCurrentLimits1().getPermanentLimit(), elementName1);
+                UcteLine ucteLine2 = new UcteLine(ucteElementId2, UcteElementStatus.REAL_ELEMENT_IN_OPERATION, (float) line.getR() / 2,
+                        (float) line.getX() / 2, (float) line.getB1(), (int) line.getCurrentLimits1().getPermanentLimit(), elementName2);
```

Comment:
I think there is an error here: maybe you have to use line.getB2() or line.getB1()+line.getB2()

Same question about the currentLimits.